shared/src/main/java/uk/gov/di/authentication/shared/entity/MFAMethod.java

```diff
@@ -111,6 +179,20 @@ public MFAMethod withUpdated(String updated) {
         return this;
     }
 
+    @DynamoDbAttribute(ATTRIBUTE_MFA_IDENTIFIER)
+    public Integer getMfaIdentifier() {
+        return mfaIdentifier;
+    }
+
+    public void setMfaIdentifier(int mfaIdentifier) {
+        this.mfaIdentifier = mfaIdentifier;
+    }
+
+    public MFAMethod withMfaIdentifier(int mfaIdentifier) {
+        this.mfaIdentifier = mfaIdentifier;
+        return this;
+    }
+
     AttributeValue toAttributeValue() {
```

Comment:
please add the new fields in here as this is used by update email to copy the user

Reply:
Ah good spot, I've added it to equals in the relevant commits. This toAttributeValue method is unused (or at least my editor is telling me it is) - can I remove it instead or is it used in some sneak way that my editor is not picking up?

Reply:
I've updated the update email integration tests to check explicitly (see last three commits here), and they pass without toAttributeValue, so I still think it's unused?

Reply:
Suggest we remove this method until we find where its needed.  If we ever have to re-instate then a unit test illustrating what its used for might be a good way to record this piece of knowledge as its not clear from the code or documentation what it could be used by.